mlir/utils/jenkins/Jenkinsfile

```diff
@@ -427,12 +429,18 @@ pipeline {
                choices: ['default', 'mfma', 'navi21', 'navi3x', 'vanilla'],
                description: 'Choose the codepath to test')
         // option to disable navi21 cells in case nodes are offline
-        booleanParam(name: 'disableNavi21', defaultValue: false,
```

Comment:
Second controversial part:  cut out navi21 by disabling it rather than removing the code, in case we have some navi21-specific bug and need to verify it.